### PR TITLE
Feature/unity docs

### PR DESCRIPTION
**Pull request by dhoncrisley-alive**
## Add docs to version 1.2.x

image

### fixes
- fix some broken routes
- fix wrong initialization disclaimer on version 1.1.x

### CODE REVIEW

**Review by hitaloloose — APPROVED** [review]

File: docs/api/unity/ad_server/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: AdServer
+sidebar_position: 1
+---
+
+static Class for handle SDK initialization and user profile settings.
```
Comment (ghost) [inline docs/api/unity/ad_server/index.md]: static Class for **handling** SDK Initialization and user profile settings.

não seria mais "correto" ?

File: docs/api/unity/ad_server/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Static Methods
+
+### `Initialize(OnInit)`
+
+Initialize the SDK
```
Comment (ghost) [inline docs/api/unity/ad_server/index.md]: **Initializes** the SDK

File: docs/api/unity/ad_server/index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+| Param  | Type            | Description                                  |
+| ------ | --------------- | -------------------------------------------- |
+| OnInit | [System.Action] | AdServer initialize event listener interface |
+
+**Return:** void.
+
+### `isInitialized` _{ get; }_
+
+Boolean that indicates if the is initialized or not.
```
Comment (ghost) [inline docs/api/unity/ad_server/index.md]: Boolean indicating whether the SDK is initialized or not.

File: docs/api/unity/ad_server/index.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+**type:** boolean.
+
+### `clientProfile` _{ get; }_
+
+Current ClientProfile object,
+you can use this for refine user's ad experience
```
Comment (ghost) [inline docs/api/unity/ad_server/index.md]: Current ClientProfile object, you can use this to refine the user's ad experience.

File: docs/api/unity/banner/index.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+### `AdView(context, unitId, size, orientation, position)`
+
+**Params:**
+
+| Param       | Type                                     | Description                                                                     |
+| ----------- | ---------------------------------------- | ------------------------------------------------------------------------------- |
+| context     | Context                                  | Parent view                                                                     |
+| unitId      | String                                   | Registered banner ad unit_id on [AdServer panel](https://adserver.adgrowth.com) |
+| size        | [AdSize](../enums/ad_size)               | Banners' dimensions size                                                        |
```
Comment (ghost) [inline docs/api/unity/banner/index.md]: não seria **banner's** ?

File: docs/api/unity/banner/index.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+| orientation | [AdOrientation](../enums/ad_orientation) | Banners' orientation                                                            |
+| x           | int                                      | Banners' horizontal position on screen                                          |
+| y           | int                                      | Banners' vertical position on screen                                            |
+
+### `Load()`
+
+Put Banner on screen and start loading
```
Comment (ghost) [inline docs/api/unity/banner/index.md]: Display the banner on the screen and start loading.

File: docs/api/unity/banner/index.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+### `Reload()`
+
+Force refresh banner
+
+**Return:** void.
+
+### `IsLoaded()`
+
+Return a boolean that indicates if the ad is loaded or not.
```
Comment (ghost) [inline docs/api/unity/banner/index.md]: Return a boolean indicating whether the ad is loaded.

File: docs/api/unity/banner/index.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+**Return:** boolean.
+
+### `IsFailed()`
+
+Return a boolean that indicates if the failed to load an ad.
```
Comment (ghost) [inline docs/api/unity/banner/index.md]: Return a boolean indicating whether the ad failed to load.

**Review by ghost — APPROVED** [review]

**Pull request merged**